frontend/javascripts/oxalis/view/left-border-tabs/modals/downsample_volume_modal.tsx

```diff
@@ -29,7 +29,7 @@ export default function DownsampleVolumeModal({
       footer={null}
       width={800}
       maskClosable={false}
-      visible
+      open
     >
```

Comment:
There was a warning that the property `visible` will be renamed to `open` and the renaming would be enforced by the next major version of antd. That's why I already changed this here.